### DISCUSSION TITLE
Why Safari extension hasn't been updated for so long?

### DISCUSSION THREAD

**Discussion by WoodpeckerBaby** [darkreader/darkreader#10445] [Q&A]
It's so outdated. Many websites don't work. Make it inline with iOS and Chrome. Break legacy support is fine.

**Comment by Gusted** [comment]
CC @alexanderby

**Comment by WoodpeckerBaby** [comment]
??

**Reply by Gusted** [reply]
I'm sorry, but I don't maintain the Safari extension. @alexanderby maintains that.

**Comment by alexanderby** [comment]
I will have a vacation between Dec 26-Jan 3, it will be a nice time for the update.

**Reply by Zabriskije** [reply]
Hey @alexanderby, any ETA on a new Safari version? Just custom colors and dark mode detection would be much appreciated. iCloud sync would be awesome.

**Reply by WoodpeckerBaby** [reply]
His personal website linked on GitHub is also down. So, it's very likely that he's no longer with us.

**Reply by alexanderby** [reply]
Hey guys, I'm fine. I'm sorry it takes so long. Things didn't go as planned with the Safari extension. Unfortunately there appeared many extensions based on Dark Reader, and the revenue has become low. Any mentions of Apple's app store make me nauseous. I returned to a normal life, having a 9-5 job and started looking after my health and family.

There was a problem preserving the settings from the previous Safari extension version, and at the same time being able to share them across iOS/macOS, that's why the release got stuck. I will take a look what I can do today, if nothing, then I will just push an update that will drop the obsolete settings.

**Reply by Zabriskije** [reply]
I don't want to be rude, I'm just genuinely curious: why revenue has been pointed out for the Safari extension? Considering that the Chrome extension it's free.

For the App Store nausea, I would suggest ditching it, but then there would be no iOS app. That could be an option when (and especially if) Apple implements sideloading.

**Reply by WoodpeckerBaby** [reply]
@alexanderby If you are no longer committed to the iOS project, it's better to hand this off to another maintainer. Please kindly discuss this with the core developers considering that you are using the default project name and is concerned official and affiliated.

**Comment by alexanderby** [comment]
The update should be available starting from today. If you have any issues during the migration, please see some notes [here](https://darkreader.org/safari/).

Unfortunately the settings will not be migrated.
